<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SLAU</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Порядок рівняння:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Метод розв'язку</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioM">
        <property name="text">
         <string>Матричний</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioR">
        <property name="text">
         <string>Обертання</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLUP">
        <property name="text">
         <string>&amp;LUP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="slvBtn">
     <property name="text">
      <string>Розв'язати</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QTableWidget" name="STable">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
